app/controllers/sessions_controller.rb

```diff
@@ -2,15 +2,16 @@ class SessionsController < ApplicationController
   def new
     redirect_to home_path if logged_in?
   end
+
   def create
     user = User.find_by(email: params[:email])
```

Comment:
I would add 1 more empty line in between this and the `if` block. Makes it more clear.